### PR TITLE
Add alert to python rest api

### PR DESCRIPTION
**Pull request by SH118415**
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-ppc64le-tests

### CODE REVIEW

**Comment by lebauce** [comment]
Thank you @SH118415
Could you please also add tests for this ?

**Comment by lebauce** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by lebauce** [comment]
@SH118415 You can call pep8 and flake8 locally to avoid most of the Python syntax errors.
You can also run the python test suite using this https://github.com/skydive-project/skydive/blob/master/scripts/ci/run-python-tests.sh#L11-L24

**Comment by SH118415** [comment]
> @SH118415 You can call pep8 and flake8 locally to avoid most of the Python syntax errors.
> You can also run the python test suite using this https://github.com/skydive-project/skydive/blob/master/scripts/ci/run-python-tests.sh#L11-L24

Sure

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

File: contrib/python/api/tests/tests.py [hunk 1]
```diff
@@ -147,6 +147,30 @@ def create_node(protocol):
...
+                                scheme=self.schemeHTTP,
+                                username=self.username,
+                                password=self.password,
+                                insecure=True)
+
+        alert1 = restclient.alert_create(
+            "https://localhost:8081",
+            "G.V().Has('Name', 'alert-ns-webhook', 'Type', 'netns')")
+
+        time.sleep(1)
```
Comment (lebauce) [inline contrib/python/api/tests/tests.py]: This is not required

Comment (SH118415) [reply contrib/python/api/tests/tests.py]: Sure i will make the changes

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by lebauce** [comment]
@SH118415 Can you please squash the 2 commits into 1 ?

**Comment by SH118415** [comment]
> @SH118415 Can you please squash the 2 commits into 1 ?

Done

**Comment by lebauce** [comment]
run skydive-python-tests

**Pull request merged**